web/yo/app/views/curationQueue.html

```diff
@@ -102,26 +102,23 @@ <h3 class="headerStyle">Curation Queue
             </tbody>
         </table>
         <div class="row" ng-show="userRole === 8">
-            <br/><br/>
-            <div class="col-sm-2"><h5>Notify Curators: </h5></div>
-            <div class="col-sm-6">
-                <select chosen multiple
-                        ng-model="curatorsToNotify"
-                        ng-options="m for m in curatorNotificationList"
-                        ng-change="prepareSendEmail()"
-                        class="chosen-select">
+            <div class="col-md-8 col-sm-12">
+                <span><b>Notify Curators:</b></span>
+                <select id="curationQueueEmailCuratorSelect"
+                        chosen multiple
+                        ng-model="data.curatorsToNotify"
+                        ng-options="m for m in data.curatorNotificationList"
+                        ng-change="prepareSendEmail()">
                 </select>
-            </div>
-            <div class="col-sm-2">
-                <button tyle="button" class="btn btn-success"
-                        ng-disabled="!curatorsToNotify || curatorsToNotify.length === 0 || email.status.sending || email.returnMessage"
-                        ng-click="sendEmail(curatorsToNotify)">
+                <button type="button" class="btn btn-sm btn-success"
+                        ng-disabled="!data.curatorsToNotify || data.curatorsToNotify.length === 0 || email.status.sending || email.returnMessage"
```

Comment:
we should remove email.returnMessage in the or statement. It never get executed.